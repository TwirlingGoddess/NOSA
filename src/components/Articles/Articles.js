import React, { Component } from 'react';

export default class Articles extends Component {
  render() {
    return(
      <div>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
        <article>
          <h2></h2>
          <img alt='' src=''/>
          <p></p>
        </article>
      </div>
    )
  }
}