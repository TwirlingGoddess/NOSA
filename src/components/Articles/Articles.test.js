import React from 'react';
import { shallow } from 'enzyme';
import Articles from './Articles';

describe('Articles', () => {
  const wrapper = shallow(<Articles />);

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
