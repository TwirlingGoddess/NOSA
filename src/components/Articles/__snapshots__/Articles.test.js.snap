// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Articles />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <article>
          <h3>
                    Why We Don't Fall Off the Edge of Our Planet
          </h3>
          <img
                    alt=""
                    src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc="
          />
          <Link
                    replace={false}
                    to="https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html"
          >
                    Read Full Story
          </Link>
</article>,
        <article>
          <h3>
                    Flat-Earther Blasts-Off in Homemade Rocket
          </h3>
          <img
                    alt=""
                    src="https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn"
          />
          <Link
                    replace={false}
                    to="https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html"
          >
                    Read Full Story
          </Link>
</article>,
        <article>
          <h3>
                    Millennials Are Confused About This Fact
          </h3>
          <img
                    alt=""
                    src="https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
          />
          <Link
                    replace={false}
                    to="https://www.livescience.com/62220-millennials-flat-earth-belief.html"
          >
                    Read Full Story
          </Link>
</article>,
        <article>
          <h3>
                    Measuring the Universe with A Flat Earth
          </h3>
          <img
                    alt=""
                    src="https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg"
          />
          <Link
                    replace={false}
                    to="https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth"
          >
                    Read & Practice
          </Link>
</article>,
        <article>
          <h3>
                    Flat-Earthers Don't Reject Climate Science
          </h3>
          <img
                    alt=""
                    src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
          />
          <Link
                    replace={false}
                    to="https://www.livescience.com/63470-flat-earth-climate-science.html"
          >
                    Read Full Story
          </Link>
</article>,
        <article>
          <h3>
                    Flat Earth Games
          </h3>
          <img
                    alt=""
                    src="https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964"
          />
          <Link
                    replace={false}
                    to="https://www.flatearth.games/blogs/news"
          >
                    Play Today
          </Link>
</article>,
      ],
      "className": "Articles",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Why We Don't Fall Off the Edge of Our Planet
</h3>,
            <img
              alt=""
              src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc="
/>,
            <Link
              replace={false}
              to="https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html"
>
              Read Full Story
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Why We Don't Fall Off the Edge of Our Planet",
            },
            "ref": null,
            "rendered": "Why We Don't Fall Off the Edge of Our Planet",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc=",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Read Full Story",
              "replace": false,
              "to": "https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html",
            },
            "ref": null,
            "rendered": "Read Full Story",
            "type": [Function],
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Flat-Earther Blasts-Off in Homemade Rocket
</h3>,
            <img
              alt=""
              src="https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn"
/>,
            <Link
              replace={false}
              to="https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html"
>
              Read Full Story
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Flat-Earther Blasts-Off in Homemade Rocket",
            },
            "ref": null,
            "rendered": "Flat-Earther Blasts-Off in Homemade Rocket",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Read Full Story",
              "replace": false,
              "to": "https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html",
            },
            "ref": null,
            "rendered": "Read Full Story",
            "type": [Function],
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Millennials Are Confused About This Fact
</h3>,
            <img
              alt=""
              src="https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
/>,
            <Link
              replace={false}
              to="https://www.livescience.com/62220-millennials-flat-earth-belief.html"
>
              Read Full Story
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Millennials Are Confused About This Fact",
            },
            "ref": null,
            "rendered": "Millennials Are Confused About This Fact",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc=",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Read Full Story",
              "replace": false,
              "to": "https://www.livescience.com/62220-millennials-flat-earth-belief.html",
            },
            "ref": null,
            "rendered": "Read Full Story",
            "type": [Function],
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Measuring the Universe with A Flat Earth
</h3>,
            <img
              alt=""
              src="https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg"
/>,
            <Link
              replace={false}
              to="https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth"
>
              Read & Practice
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Measuring the Universe with A Flat Earth",
            },
            "ref": null,
            "rendered": "Measuring the Universe with A Flat Earth",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Read & Practice",
              "replace": false,
              "to": "https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth",
            },
            "ref": null,
            "rendered": "Read & Practice",
            "type": [Function],
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Flat-Earthers Don't Reject Climate Science
</h3>,
            <img
              alt=""
              src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
/>,
            <Link
              replace={false}
              to="https://www.livescience.com/63470-flat-earth-climate-science.html"
>
              Read Full Story
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Flat-Earthers Don't Reject Climate Science",
            },
            "ref": null,
            "rendered": "Flat-Earthers Don't Reject Climate Science",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc=",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Read Full Story",
              "replace": false,
              "to": "https://www.livescience.com/63470-flat-earth-climate-science.html",
            },
            "ref": null,
            "rendered": "Read Full Story",
            "type": [Function],
          },
        ],
        "type": "article",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Flat Earth Games
</h3>,
            <img
              alt=""
              src="https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964"
/>,
            <Link
              replace={false}
              to="https://www.flatearth.games/blogs/news"
>
              Play Today
</Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Flat Earth Games",
            },
            "ref": null,
            "rendered": "Flat Earth Games",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Play Today",
              "replace": false,
              "to": "https://www.flatearth.games/blogs/news",
            },
            "ref": null,
            "rendered": "Play Today",
            "type": [Function],
          },
        ],
        "type": "article",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <article>
            <h3>
                        Why We Don't Fall Off the Edge of Our Planet
            </h3>
            <img
                        alt=""
                        src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc="
            />
            <Link
                        replace={false}
                        to="https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html"
            >
                        Read Full Story
            </Link>
</article>,
          <article>
            <h3>
                        Flat-Earther Blasts-Off in Homemade Rocket
            </h3>
            <img
                        alt=""
                        src="https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn"
            />
            <Link
                        replace={false}
                        to="https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html"
            >
                        Read Full Story
            </Link>
</article>,
          <article>
            <h3>
                        Millennials Are Confused About This Fact
            </h3>
            <img
                        alt=""
                        src="https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
            />
            <Link
                        replace={false}
                        to="https://www.livescience.com/62220-millennials-flat-earth-belief.html"
            >
                        Read Full Story
            </Link>
</article>,
          <article>
            <h3>
                        Measuring the Universe with A Flat Earth
            </h3>
            <img
                        alt=""
                        src="https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg"
            />
            <Link
                        replace={false}
                        to="https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth"
            >
                        Read & Practice
            </Link>
</article>,
          <article>
            <h3>
                        Flat-Earthers Don't Reject Climate Science
            </h3>
            <img
                        alt=""
                        src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
            />
            <Link
                        replace={false}
                        to="https://www.livescience.com/63470-flat-earth-climate-science.html"
            >
                        Read Full Story
            </Link>
</article>,
          <article>
            <h3>
                        Flat Earth Games
            </h3>
            <img
                        alt=""
                        src="https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964"
            />
            <Link
                        replace={false}
                        to="https://www.flatearth.games/blogs/news"
            >
                        Play Today
            </Link>
</article>,
        ],
        "className": "Articles",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Why We Don't Fall Off the Edge of Our Planet
</h3>,
              <img
                alt=""
                src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc="
/>,
              <Link
                replace={false}
                to="https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html"
>
                Read Full Story
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Why We Don't Fall Off the Edge of Our Planet",
              },
              "ref": null,
              "rendered": "Why We Don't Fall Off the Edge of Our Planet",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS81OTgvb3JpZ2luYWwvZmxhdC1lYXJ0aC1OTy1SRVVTRS5qcGc=",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Read Full Story",
                "replace": false,
                "to": "https://www.livescience.com/62454-flat-earthers-explain-pac-man-effect.html",
              },
              "ref": null,
              "rendered": "Read Full Story",
              "type": [Function],
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Flat-Earther Blasts-Off in Homemade Rocket
</h3>,
              <img
                alt=""
                src="https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn"
/>,
              <Link
                replace={false}
                to="https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html"
>
                Read Full Story
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Flat-Earther Blasts-Off in Homemade Rocket",
              },
              "ref": null,
              "rendered": "Flat-Earther Blasts-Off in Homemade Rocket",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://img.purch.com/h/1400/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8wNTUvb3JpZ2luYWwvTWlrZS1IdWdoZXMtTk8tUkVVU0UuanBn",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Read Full Story",
                "replace": false,
                "to": "https://www.livescience.com/62115-flat-earther-mike-hughes-launches-homemade-rocket.html",
              },
              "ref": null,
              "rendered": "Read Full Story",
              "type": [Function],
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Millennials Are Confused About This Fact
</h3>,
              <img
                alt=""
                src="https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
/>,
              <Link
                replace={false}
                to="https://www.livescience.com/62220-millennials-flat-earth-belief.html"
>
                Read Full Story
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Millennials Are Confused About This Fact",
              },
              "ref": null,
              "rendered": "Millennials Are Confused About This Fact",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://img.purch.com/rc/360x240/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5OS8yMzQvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc=",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Read Full Story",
                "replace": false,
                "to": "https://www.livescience.com/62220-millennials-flat-earth-belief.html",
              },
              "ref": null,
              "rendered": "Read Full Story",
              "type": [Function],
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Measuring the Universe with A Flat Earth
</h3>,
              <img
                alt=""
                src="https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg"
/>,
              <Link
                replace={false}
                to="https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth"
>
                Read & Practice
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Measuring the Universe with A Flat Earth",
              },
              "ref": null,
              "rendered": "Measuring the Universe with A Flat Earth",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://ka-perseus-images.s3.amazonaws.com/af5238aabdddaeb02aeae246a605286dde1ad743.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Read & Practice",
                "replace": false,
                "to": "https://www.khanacademy.org/partner-content/nasa/measuringuniverse/measure-the-solarsystem/a/a-flat-earth",
              },
              "ref": null,
              "rendered": "Read & Practice",
              "type": [Function],
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Flat-Earthers Don't Reject Climate Science
</h3>,
              <img
                alt=""
                src="https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc="
/>,
              <Link
                replace={false}
                to="https://www.livescience.com/63470-flat-earth-climate-science.html"
>
                Read Full Story
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Flat-Earthers Don't Reject Climate Science",
              },
              "ref": null,
              "rendered": "Flat-Earthers Don't Reject Climate Science",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ni84NDMvb3JpZ2luYWwvZmxhdC1lYXJ0aC5qcGc=",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Read Full Story",
                "replace": false,
                "to": "https://www.livescience.com/63470-flat-earth-climate-science.html",
              },
              "ref": null,
              "rendered": "Read Full Story",
              "type": [Function],
            },
          ],
          "type": "article",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Flat Earth Games
</h3>,
              <img
                alt=""
                src="https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964"
/>,
              <Link
                replace={false}
                to="https://www.flatearth.games/blogs/news"
>
                Play Today
</Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Flat Earth Games",
              },
              "ref": null,
              "rendered": "Flat Earth Games",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "",
                "src": "https://cdn.shopify.com/s/files/1/0017/8887/0729/articles/FEresearcher_1512x.jpg?v=1532986964",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Play Today",
                "replace": false,
                "to": "https://www.flatearth.games/blogs/news",
              },
              "ref": null,
              "rendered": "Play Today",
              "type": [Function],
            },
          ],
          "type": "article",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
